src/modules/commander/Commander.cpp

```diff
@@ -1480,15 +1480,15 @@ Commander::run()
 			param_get(_param_ef_current2throttle_thres, &ef_current2throttle_thres);
 			param_get(_param_ef_time_thres, &ef_time_thres);
 			param_get(_param_geofence_action, &geofence_action);
-			param_get(_param_disarm_land, &disarm_when_landed);
+			param_get(_param_disarm_land, &disarm_when_landed_timeout);
```

Comment:
Can you move this to the new param wrapper while you're at it? See Commander.hpp

Reply:
sure, makes sense.

Reply:
done 👍

Reply:
Thanks